### DISCUSSION TITLE
Dump different sorts of configs

### DISCUSSION THREAD

**Discussion by csuno** [netbox-community/netbox#13991] [Q&A]
Hy, i pretty new with netbox... so please be patient ;)

I use Netbox-Docker 3.6.3.

I try to dump my running config from several hpe aruba, cisco, and other brands into the DCIM/Device objects in Netbox.
Is there a designated way for such a common task? Or should i create a "Custom Field" for this?

There are several plugins out there, but i don't know, if they will fetch all of my devices? But features like "live-data", syncing, versions and diff-control are pretty nice to have!

Thanks for your feedback!

**Comment by chris240189** [comment]
I have been testing with https://github.com/miaow2/netbox-config-diff and run diffs of templated config against a folder of config files (netbox data-source), that is pulled from devices via oxidized.
